modules/control/common/mrac_controller.cc

```diff
@@ -19,23 +19,33 @@
 #include <cmath>
 #include <vector>
 
+#include "Eigen/LU"
+
 #include "cyber/common/log.h"
 
 namespace apollo {
 namespace control {
 
-double MracController::Control(const double command, const double state,
+using Matrix = Eigen::MatrixXd;
+
+double MracController::Control(const double command, const Matrix state,
                                const double dt) {
-  // check if the reference model well set up during the initilization
+  // check if the reference/adaption model well set up during the initilization
   if (!reference_model_enabled_) {
-    TransformReferenceModel(dt);
```

Comment:
Don not build reference/adaption model in `::Control(...)`, it should only be done once in `Init()` and after that only update the model if needed.

Reply:
Fixed per comment. Thanks

Reply:
merge these two `if` please

Reply:
Fixed